Assets/Scripts/InputManager.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public class InputManager : MonoBehaviour
+{
+    private static InputManager _instance;
+    public List<SourceBuildingAsset> buildingList = new List<SourceBuildingAsset>();
+    public UnityEvent CreateBuilding = new UnityEvent();
```

Comment:
why are you initialising it this way? Not that it really matters but like is this rider or you?

Reply:
i was getting nullreferences so i tried this, but yeag it can be removed